core/templates/dev/head/pages/maintenance/maintenance.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html ng-app="oppia" lang="<[currentLang]>" ng-controller="Maintenance" itemscope itemtype="http://schema.org/Organization">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
+    <title itemprop="name" translate="Down for Maintenance - Oppia"></title>
+    {% include 'pages/header_css_libs.html' %}
+
+    <script>
+      var GLOBALS = {
```

Comment:
As mentioned offline with Sean, the approach is only to keep a minimal set of Globals for the maintenance page to run (ideally none).